app-config.yaml

```diff
@@ -101,37 +121,22 @@ catalog:
         #   token:
         #     $secret:
         #       env: GHE_PRIVATE_TOKEN
-    bitbucketApi:
-      username:
-        $secret:
-          env: BITBUCKET_USERNAME
-      appPassword:
-        $secret:
-          env: BITBUCKET_APP_PASSWORD
-    gitlabApi:
-      privateToken:
-        $secret:
-          env: GITLAB_PRIVATE_TOKEN
-    azureApi:
-      privateToken:
-        $secret:
-          env: AZURE_PRIVATE_TOKEN
 
   locations:
     # Backstage example components
-    - type: github
+    - type: url
```

Comment:
Would be nice to make type optional later - or even make the type be string-or-object-with-optional-type

Reply:
Yep, intending to make that happen later after all of this migration :>